khelo/lib/ui/flow/team/add_team_member/add_team_member_screen.dart

```diff
@@ -105,184 +95,141 @@ class AddTeamMemberScreen extends ConsumerWidget {
-                    ? context.l10n.add_team_member_added_text
-                    : context.l10n.common_add_title.toUpperCase()),
-              ),
-            ),
-          ],
-        ));
-  }
-
   Widget _selectedPlayerList(
     BuildContext context,
-    AddTeamMemberViewNotifier notifier,
     AddTeamMemberState state,
   ) {
     return SizedBox(
       height: 60,
-      child: Stack(
-        children: [
-          ListView.separated(
-              scrollDirection: Axis.horizontal,
-              padding: const EdgeInsets.only(right: 100),
-              itemBuilder: (context, index) {
-                final user = state.selectedUsers[index];
-                return SizedBox(
-                  height: 60,
-                  width: 65,
-                  child: Stack(
-                    children: [
-                      ImageAvatar(
-                        initial: user.nameInitial,
-                        imageUrl: user.profile_img_url,
-                        size: 60,
-                      ),
-                      Align(
-                        alignment: Alignment.topRight,
-                        child: OnTapScale(
-                            onTap: () {
-                              notifier.unSelectUser(user);
-                            },
-                            child: Icon(
-                              Icons.cancel_rounded,
-                              color: context.colorScheme.textPrimary,
-                            )),
-                      ),
-                    ],
-                  ),
-                );
-              },
-              separatorBuilder: (context, index) {
-                return const SizedBox(
-                  width: 20,
-                );
-              },
-              itemCount: state.selectedUsers.length),
-        ],
+      child: ListView.separated(
+        scrollDirection: Axis.horizontal,
+        itemCount: state.selectedUsers.length,
+        itemBuilder: (context, index) {
+          final user = state.selectedUsers[index];
+          return SizedBox(
+            height: 60,
+            width: 65,
+            child: Stack(
+              children: [
+                ImageAvatar(
+                  initial: user.nameInitial,
+                  imageUrl: user.profile_img_url,
+                  size: 60,
+                ),
+                Align(
+                  alignment: Alignment.topRight,
+                  child: OnTapScale(
+                      onTap: () {
+                        notifier.unSelectUser(user);
+                      },
+                      child: Icon(
+                        Icons.cancel_rounded,
+                        color: context.colorScheme.textPrimary,
+                      )),
+                ),
+              ],
+            ),
+          );
+        },
+        separatorBuilder: (context, index) => const SizedBox(width: 20),
       ),
     );
   }
+
+  void _observeActionError() {
```

Comment:
is there any reason we moved these to bottom? we have kept them at top in every screens, if we want to change this then we must move them to bottom in every screen for consistency